CHANGELOG.md

```diff
@@ -40,6 +40,10 @@ release.
 - Add initial OpenTracing compatibility section.
   ([#1101](https://github.com/open-telemetry/opentelemetry-specification/pull/1101))
 
+### Compliance
+
+- Update Jaeger Exporter service name mapping to be supported in Go. ([#TBD])
```

Comment:
I don't believe we've added `CHANGELOG` entries for spec compliance matrix updates.  This probably isn't needed.